### ISSUE TITLE
Frame Dropping

### ISSUE THREAD

**Issue by snekyo** [fabiangreffrath/woof#1454]
Sometimes, for a few seconds, the game will drop in frames. I run Woof through steam with the non-steam game system, and I've activated the fps counter in the bottom right that steam can add. Looking at it, it doesn't actually show any frame decrease.
I'm not running any mods.

**Comment by rfomin** [comment]
Are you running Woof 12.0.2 or a development version with new menus etc (GitHub artifact)?

**Comment by snekyo** [comment]
> Are you running Woof 12.0.2 or a development version with new menus etc (GitHub artifact)?

I used Woof 12.0.2.
Somebody on the Chocolate DOOM Discord recommended try running it not on Steam, but that didn't work. They also suggested that I use the Exclusive Fullscreen option which I hadn't had switched on, though I'm not sure if it works or not seeing as I've just moved over to Crispy DOOM. I'm not too much into DOOM mods, so that's why.
